### ISSUE TITLE
Inclusion of Tests

### ISSUE THREAD

**Issue by Jayitha** [SimpleRA/SimpleRA#3]
Since we expect a lot of pull requests on the master branch, it would be nice to have tests to ensure the functionality of the system hasn't been affected as a result of the new feature being added.

**Comment by ujwal-narayan** [comment]
Hey
I'd like to work on this! 
I've already written some in Python with Pyexpect, hope that's alright! 

Thanks,
Ujwal

**Comment by Jayitha** [comment]
Yup, that should be fine, let us know how you plan to go forward and a small description of each test either in the comments or here would be nice.
